src/js/components/AppListComponent.jsx

```diff
@@ -396,92 +381,130 @@ var AppListComponent = React.createClass({
 
     var totalColumnSpan = 8;
 
+    var inlineDialog = null;
+
+    if (pageIsLoading) {
+      inlineDialog = (
+        <CenteredInlineDialogComponent title="Loading Applications..."
+          message="Please wait while applications are being retrieved." />
+      );
+    }
+
+    if (pageHasNoRunningApps) {
+      let message = "Do more with Marathon by creating and organizing " +
+        "your applications.";
+      inlineDialog = (
+        <CenteredInlineDialogComponent additionalClasses="muted"
```

Comment:
Yeah it's pretty brutal. I was in fact thinking about doing the DialogSeverity thing here too, but was waiting to get that PR merged first. Actually though, they are not related at all. In practice we only care about `error` and `muted` (judging from the mockups, at least). What do you suggest?